requirements.txt

```diff
@@ -0,0 +1,6 @@
+Django>=3.0.9,<3.1.0
```

Comment:
The range for the version is OK for libraries, but your application should have a fixed version.
Imagine, there's an update between two deploys and you don't even realize and it suddenly breaks something :wink: 

Though this is totally fine for this exercise